index.js

```diff
@@ -1,5 +1,164 @@
 const battleship = () => {
-  return 'The winner is...?'
+
+  // ## Step 1: Create Players
+
+let playerOne = {
+  name: prompt("playerOne - What is your first name?"),
+  shipCount: parseInt(4),
+  gameBoard: [
+    [0,0,0,0],
+    [0,0,0,0],
+    [0,0,0,0],
+    [0,0,0,0]
+  ]
+}
+
+let playerTwo = {
+  name: prompt("playerTwo - What is your first name?"),
+  shipCount: parseInt(4),
+  gameBoard: [
+    [0,0,0,0],
+    [0,0,0,0],
+    [0,0,0,0],
+    [0,0,0,0]
+  ]
+}
+
+//## Step 2: Randomly Add Ships to each Board
+
+//PlayerOne add ships to the existing array
+let shipPlacePlayerOne = 0;
+
+for (var i = 0; i < 4; i++) {
+
+ let xCoordinateOne = Math.floor( Math.random() * (3 - 0 + 1) + 0);
+ let yCoordinateOne = Math.floor( Math.random() * (3 - 0 + 1) + 0);
+
+ console.log (xCoordinateOne, yCoordinateOne);
+
+ if (playerOne.gameBoard[xCoordinateOne][yCoordinateOne] === 0) {
+    playerOne.gameBoard[xCoordinateOne][yCoordinateOne] = 1;
+    shipPlacePlayerOne +=1;
+ } 
+}
+// show PlayerOne X and Y coordinates
+console.log (JSON.stringify(playerOne.gameBoard));
+
+// PlyaerTwo add ships to the existing array
+let shipPlacePlayerTwo = 0;
+
```

Comment:
Your two for loops here could be moved into one function that is then called for each player.